composer.json

```diff
@@ -20,11 +20,11 @@
         }
     ],
     "require": {
-        "php": "^7.2",
-        "illuminate/database": "^6.0|^7.0"
+        "php": "^7.2|^7.3",
+        "illuminate/database": "^6.0|^7.0|^8.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^8.0",
-        "orchestra/testbench": "^5.1"
+        "phpunit/phpunit": "^8.4|^9.0",
```

Comment:
What's the reasoning behind the `^8.4` change?

Reply:
Just wanted to be consistent with Laravel framework (versions 7.x and 8.x).

Reply:
I see, makes sense!